### PR TITLE
feat: Proof of concept play sound on card flip

### PR DESCRIPTION
**Pull request by katjam**
@Nikomus This is just a proof of concept that play success sound on every card flip... but I #40 probably wants merging before I do more here.

Wanted to PR in case I am on the right track.

I'm thinking since not critical for the game to work is ok if Html Audio not supported everywhere
I'll give config of volume to user & abstract so different effects can be triggered by different events

### CODE REVIEW

File: src/Main.elm [hunk 1]
```diff
@@ -181,13 +191,24 @@ view model =
...
                 ]
             ]
         , footer [ class "page-section" ]
             [ a [ href "https://wearedarts.org.uk" ] [ text "wearedarts.org.uk" ] ]
         ]
 
 
+renderAudio : SoundEffect -> Html Msg
+renderAudio soundEffect =
+    Html.audio
```
Comment (katjam) [inline src/Main.elm:204]: I've used the Html namespace here since they are not standard components - to make it obvious which library they come from.

Comment (Nikomus) [reply src/Main.elm:204]: Looks like just a `<audio>` element rather than anything from a library?

Comment (katjam) [reply src/Main.elm:204]: Yeah there is no library apart from Elm core `Html` involved. It is just an `<audio>` element. I thought if I put it in as `audio` there might be some confusion as to whether is was just Html

**Comment by katjam** [comment]
@Nikomus Are you happy with this in principle? I'd like to build the logic on top of it if you are.

**Comment by Nikomus** [comment]
Sounds good 🎶 

Only thing to maybe think about is that keeping all of the elements in the page might get quite large. <audio> elements have a lot of shadow DOM nodes, it looks like ~150 nodes are added per click. We might want to keep the list a fixed size and replace the oldest if we're adding a lot more sound triggers.

**Comment by katjam** [comment]
Not sure how to check the shadow DOM nodes but we must be able to limit that somehow. Closing as it was a spike anyway.. but will build on for #35 and we can pair on getting the spawn per click under control.

**Pull request closed without merging**